freemocap/data_layer/export_data/generate_jupyter_notebook/generate_jupyter_notebook.py

```diff
@@ -13,14 +11,15 @@ def generate_jupyter_notebook(path_to_recording: Union[str, Path]):
     path_to_template_notebook = Path(__file__).parent / "freemocap_template.ipynb"
     path_to_output_notebook = Path(path_to_recording) / f"{recording_name}.ipynb"
 
-    logger.info(f"Jupyter notebook generated at {path_to_output_notebook}")
-    success = pm.execute_notebook(
-        input_path=path_to_template_notebook,
-        output_path=path_to_output_notebook,
-        parameters=dict(path_to_recording=str(path_to_recording)),
-    )
+    with open(path_to_template_notebook, "r") as file:
+        template_notebook = json.load(file)
+
+    template_notebook["cells"][3]["source"] = [f'path_to_recording = "{str(path_to_recording)}"']
```

Comment:
This is ok enough for now, but that hard coded `3` is a pretty brittle solution. We'll have to remember to update it when we change our ipynb template (which we should do soon!)

Reply:
Unfortunately, `template_notebook["cells"]` is a list, so we don't have many options for indexing.